src/batou_ext/oci.py

```diff
@@ -0,0 +1,115 @@
+import hashlib
+import os
+import re
+
+import pkg_resources
+from batou import UpdateNeeded
+from batou.component import Attribute, Component
+from batou.lib.file import File
+from batou.lib.service import Service
+
+import batou_ext.nix
+
+
+@batou_ext.nix.rebuild
+class Container(Component):
+    """
+    A OCI Container component.
+    With this component you can dynamically schedule docker containers to be run on the target host
+
+    Example:
+    ```
+    self += batou_ext.oci.Container("mysql", "8.0")
```

Comment:
I think this example is not complete

Reply:
This works given the custom `__init__` below (which might be the cause of confusion)